content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1294,6 +1294,18 @@ has been truncated to 1000.
 
 If the number of backend endpoints falls below 1000, the control plane removes this annotation.
 
+### endpoints.kubernetes.io/last-change-trigger-time
+
+Type: Annotation
+
+Example: `endpoints.kubernetes.io/last-change-trigger-time: 2023-07-20T04:45:21Z`
+
+Used on: Endpoints
+
+This annotation set to an [Endpoints](/docs/concepts/services-networking/service/#endpoints) object that
+represents the timestamp (stored as RFC 3339 date-time string, e.g. '2018-10-22T19:32:52.1Z'). This is timestamp
```

Comment:
(nit) We [avoid Latin phrases](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases) and their abbreviations